src/MIDebugEngine/Engine.Impl/DebuggedThread.cs

```diff
@@ -226,6 +226,18 @@ private ThreadContext CreateContext(TupleValue frame)
             ulong? pc = frame.TryFindAddr("addr");
             MITextPosition textPosition = MITextPosition.TryParse(frame);
             string func = frame.TryFindString("func");
+
+            if (this._debugger.MICommandFactory.Mode == MIMode.Lldb)
+            {
+                if (func.Contains("("))
+                {
+                    // LLDB adds the types of variables to the func. This doesn't conform 
+                    // the mi protocol and results in duplicate data when the ad7 stackframe
+                    // object does the same thing. 
+                    func = func.Substring(0, func.IndexOf('('));
```

Comment:
Is there any chance that a '(' could be part of a type name in some sort of template case?